statchatui/src/Stats/Stats.jsx

```diff
@@ -1,16 +1,28 @@
 import React from "react";
 
 const Stats = ({ statData }) => {
-  const kills = statData?.total?.kills?.value;
+  let kills, level, name, rank;
+  if (statData != null) {
+    console.log(statData);
+    kills = statData.total.kills.value;
+    level = statData.global.level;
+    name = statData.global.name;
+    rank = statData.global.rank.rankName;
+  }
+
   return (
     <div className="container">
       {statData ? (
         <div>
-          <h1>Stats:</h1>
-          <p>Kills: {kills}</p>
+          <h1>Apex Legends</h1>
+          <h2>Stats:</h2>
+          <h3>Name: {name}</h3>
+          <h3>Level: {level}</h3>
+          <h3>Rank: {rank}</h3>
+          <h3>Kills: {kills}</h3>
         </div>
       ) : (
-        <div>Loading...</div>
+        <div>Stats not yet fetched.</div>
```

Comment:
Looks good. To make this a little cleaner we could return early so we only need to check `statData` once!